src/pages/daoMember.tsx

```diff
@@ -80,6 +81,18 @@ export const DaoMember: React.FC = () => {
     {enabled: !!memberAddress && !!daoDetails}
   );
 
+  const {
+    data: memberCreatedProposals,
+    isLoading: isMemberCreatedProposalsLoading,
+  } = useCreatorProposals(
+    {
+      address: memberAddress,
+      pluginAddress: pluginAddress || '',
+      pluginType,
+    },
+    {enabled: !!memberAddress && !!daoDetails}
```

Comment:
In that way we avoid calling the hook with an empty string on the `pluginAddress` parameter which could throw some error on the console
```suggestion
      pluginAddress: pluginAddress as string,
      pluginType,
    },
    {enabled: !!memberAddress && !!daoDetails && !!pluginAddress}
```